src/modules/shared/types/media.ts

```diff
@@ -61,7 +61,7 @@ export interface MediaInfo {
 	schema_contributor: unknown | null;
 	schema_maintainer?: {
 		schema_identifier: string;
-	}[];
+	};
```

Comment:
Not a typo I'm afraid (src: POST http://localhost:3100/media/or-rf5kf25)
![image](https://user-images.githubusercontent.com/90180417/163124907-0ade477d-1d88-4ce6-bce9-35e35d41d4c4.png)

Reply:
Dat zou niet mogen 🤔  Laatste versie van proxy en hasura?
- `GET /media/{esIndex}/{id}/similar`
![image](https://user-images.githubusercontent.com/85545258/163180602-1dde0e19-fd61-43c2-9d50-0266f50eae6a.png)
- `GET media/{esIndex}`
![image](https://user-images.githubusercontent.com/85545258/163180709-3cb219d4-3861-4b79-b600-1eacaf81d866.png)

Reply:
Hmm; duidelijk niet want kan het niet meer reproduceren